packages/editor/src/components/widgets/table/cell/ExpandableCell.tsx

```diff
@@ -11,9 +11,10 @@ type ExpandableCellProps<TData> = {
   isUnknown?: boolean;
   icon?: IvyIcons;
   title?: string;
+  helperInfo?: string;
 };
 
-export function ExpandableCell<TData>({ cell, isLoaded, loadChildren, isUnknown, icon, title }: ExpandableCellProps<TData>) {
+export function ExpandableCell<TData>({ cell, isLoaded, loadChildren, isUnknown, icon, title, helperInfo }: ExpandableCellProps<TData>) {
```

Comment:
don't really like the name helperinfo. additionalInfo or something like that is more suitable I think
And helperinfo is really near to the helper text in the browser